postgres/codenamemap.yaml

```diff
@@ -40,13 +46,19 @@
   {% if repo.use_upstream_repo == true %}
     {% set fromrepo = repo.fromrepo|default(name ~ '-pgdg', true) %}
     {% set version = repo.version %}
+    {% set conf_dir = '/etc/postgresql/' + version|string + '/main' %}
+    {% set data_dir = '/var/lib/postgresql/' + version|string + '/main' %}
```

Comment:
Is this really applicable to Fedora? I'm not sure.

Reply:
I checked my commits on #187 (which was tested heavily) and you're correct. Not needed. Fixed.